packages/lesswrong/components/localGroups/CommunityHome.tsx

```diff
@@ -65,7 +65,8 @@ const CommunityHome = ({classes}: {
 
   const isEAForum = forumTypeSetting.get() === 'EAForum';
   const isAdmin = userIsAdmin(currentUser);
-  const canCreateEvents = currentUser && (!isEAForum || isAdmin);
+  const canCreateEvents = currentUser;
+  const canCreateGroups = currentUser && (!isEAForum || isAdmin);
```

Comment:
@jpaddison3 I did make it so that forum users can all create events from the main Community page. My concern was that people who are group organizers would see this button on the main page and start using that instead of navigating into their group first, and not realize that they could not assign their group from the form.

Now when a user clicks on the "Create New Event" button from the main Community page, they get the default form which does **not** put the event in a group, but they can select a group from the dropdown if they want.

Reply:
There's also a create event button on the user menu dropdown, which is what I thought was the thing we'd open up. I'd also open up that while you're at it.

Reply:
Oh thanks, I didn't realize there were more places to look - I enabled it from that menu, and also in the legend on the Community map.